homeassistant/helpers/template.py

```diff
@@ -2619,6 +2640,12 @@ def wrapper(_: Any, *args: _P.args, **kwargs: _P.kwargs) -> _R:
         self.globals["device_id"] = hassfunction(device_id)
         self.filters["device_id"] = self.globals["device_id"]
 
+        self.globals["issues"] = hassfunction(issues)
+        self.filters["issues"] = self.globals["issues"]
```

Comment:
How is this useful as a filter?

Reply:
Unsure what you mean, but this format is kept in line with the others, e.g. `areas` and `area_id`.

Reply:
adding this to `filters` means the `issues` function can be called by using Jinja's [filter syntax](https://jinja.palletsprojects.com/en/3.1.x/templates/#filters), `|`

I don't see how this is useful for the `issues` function, an I don't think it's useful for `areas` either.

Reply:
Ah, got it. Thanks for clarifying 👍